prot.c

```diff
@@ -932,7 +938,8 @@ fmt_stats(char *buf, size_t size, void *x)
             srv->wal.nrec,
             srv->wal.filesize,
             id,
-            node_info.nodename);
+            node_info.nodename,
+            srv->drain_mode);
```

Comment:
Apologies; I didn't explain my feedback clearly enough the first time.

> How about `drain_mode ? "true" : "false"` here instead?

This is a suggestion to use the expression here, on this line, instead of `srv->drain_mode`. We already store the information about whether we're in drain mode in the global variable drain_mode, so there's no need to store it also in a field on srv. In fact, storing it redundantly is extra complexity that increases the chances of a bug in the future. It's better to compute the value here, directly from the boolean flag.

One way to do that is with a ternary expression here on this line. It's not the only way, but I recommend it because it's simple and straightforward.

> And I suggest putting this before the id field instead of after, so there's less noise in the diff.

It seems like you chose to ignore this suggestion. If you disagree with it that's fine, I'd just like to know why. Is there a particular reason to put the drain mode last in the list? If not, it would be marginally better somewhere else.

Putting this all together, we would get:

```
...
srv->wal.nrec,
srv->wal.filesize,
id,
drain_mode ? "true" : "false",
node_info.nodename);
```